### DISCUSSION TITLE
Solution benchmarks

### DISCUSSION THREAD

**Discussion by lesnyrysiek** [codewars/codewars.com#2840] [Ideas]
While voting for best practice sometimes can be quite subjective and some non-grouped solutions may look similar, but sometimes small details might make a difference. If solutions include benchmarks, then it might would be easier to compare similar looking solutions.

**Comment by hobovsky** [comment]
Please see [this discussion](https://github.com/codewars/codewars.com/issues/959) and other, linked discussions, and especially [this post](https://github.com/codewars/codewars.com/issues/959#issuecomment-613094011) for some information why the feature of benchamrking is not, and is not likely to be, implemented.
